### PR TITLE
initial fastlane structures

### PR DESCRIPTION
**Pull request by IzzySoft**
Helpful if you should apply for listing with [F-Droid](https://f-droid.org/), useful also in conjunction with your app listing [on my repo](https://apt.izzysoft.de/fdroid/index/apk/org.traccar.manager) – and (with the Fastlane binaries) as well for deployment to Google Play: one place to rule them all :smiley:

You might wish to add your images (featureGraphic, screenshots) as well as per-release changelogs. My [Fastlane cheat-sheet](https://gitlab.com/snippets/1895688) can guide you with that.

### CODE REVIEW

**Comment by IzzySoft** [comment]
If anything feels wrong or missing, please let me know @tananaev :wink: I'm currently unsure if this PR was missed, or something is keeping you from processing it…

**Comment by tananaev** [comment]
I haven't missed it, but I'm not really sure we need fastlane. F-Droid can automatically pick up releases from GitHub, as far as I know.

**Comment by IzzySoft** [comment]
> F-Droid can automatically pick up releases from GitHub

But not the release notes (disclosure: I'm one of the F-Droid maintainers). We need fastlane to show screenshots, for example. And we much prefer it getting summary/description the very same way – as that way the authors of the apps can maintain it without needing extra MRs for every change. Besides, the same is true for my repo :wink:

So: It might not be essential, but strongly recommended. Which is why I provided you with a "starter package" you can built upon, containing little more than the essentials (summary+description are the minimum).

**Comment by tananaev** [comment]
I get it, but there's also maintenance cost and learning curve for us. So I'm not sure if it's worth it, but let's keep the issue open for now in case we decide to implement it later.

**Comment by IzzySoft** [comment]
I understand – which is why I kept it minimal, and didn't go for the "full feature set":

* short description (aka "summary") and title will be rather static and rarely change
* same for the icon – which in case of a change is easily replaced
* full description (aka "description") is the one which will change with the app evolving – but usually not that frequently. If you have no German speakers on your team to maintain that translation, you could always drop `de/full_description.txt` when `en-US/full_description.txt` is being reworked. This would still meet the basic requirements (which is: summary+description for en-US)
* screenshots would be nice to have, and again fully sufficient in `en-US` – which is always used as fallback for elements missing with other locales
* all else is "icing on the cake" – though per-release changelogs are often asked for by users.

TL;DR: Maintenance costs are rather low IMHO, or can be kept low. I've been providing similar "starter sets" to many other projects which never had used fastlane before (or even heard about it for the first time by means of my PR). The basic idea is easy. And if you forget, others like me certainly will help with PRs :wink: That's something not even needing development skills, so it can be delegated to "beginners" who would like to contribute but have no experience yet.

Oh, and should you have questions, feel free to ask them – I'll help if I can. I'm the one responsible for "metadata" at F-Droid, so it's in my own interest as well :smile:

**Comment by IzzySoft** [comment]
@tananaev any chance this will be processed? I'm about cleaning up my Github account before this year's end and drop "old forks" I don't plan to maintain – and once I drop the fork of your app, this PR will go "stale" as the source is gone.

**Comment by tananaev** [comment]
I don't think we are ready to merge it at this time.

**Comment by IzzySoft** [comment]
Well, it's up to you. Though I do not understand what stands in its way. It doesn't conflict with anything, sitting in its own tree. I won't push you of course, this is just an offer. But please understand that I need to make some "house keeping" as well to keep my things organized.

On the other hand: once my fork is gone, you won't need to get ready to merge this anymore :speak_no_evil: :laughing:

**Comment by IzzySoft** [comment]
I'm dropping my fork now – it's been half a year. This PR will remain mergeable, as I just found out – so timing is completely up to you now. I still do not understand the hesitation (there are no conflicts at all, it does not cross with any of your code, so as I see it it only adds value without any disadvantages) – but well…
